starknet_py/utils/typed_data_test.py

```diff
@@ -134,18 +152,18 @@ def test_message_hash(example, account_address, msg_hash):
 
 domain_type_v0 = {
     "StarkNetDomain": [
-        {"name": "name", "type": "felt"},
-        {"name": "version", "type": "felt"},
-        {"name": "chainId", "type": "felt"},
+        Parameter(name="name", type="felt"),
+        Parameter(name="version", type="felt"),
+        Parameter(name="chainId", type="felt"),
     ]
 }
 
 domain_type_v1 = {
     "StarknetDomain": [
-        {"name": "name", "type": "shortstring"},
-        {"name": "version", "type": "shortstring"},
-        {"name": "chainId", "type": "shortstring"},
-        {"name": "revision", "type": "shortstring"},
+        Parameter(name="name", type="shortstring"),
+        Parameter(name="version", type="shortstring"),
+        Parameter(name="chainId", type="shortstring"),
+        Parameter(name="revision", type="shortstring"),
     ]
 }
```

Comment:
Update list of `dict` to list of `Parameter`, according to `TypedData.types` type [ref](https://github.com/software-mansion/starknet.py/blob/a59c8a278c986002efda9e0c64afe2d46086f3fe/starknet_py/utils/typed_data.py#L85).